rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/charts/MemoryLineChart.java

```diff
@@ -51,15 +51,12 @@ public MemoryLineChart(RMController controller, String jmxServerUrl) {
               "sigar:Type=Mem",
               new String[] { "ActualUsed", "ActualFree", "Total" },
               "Memory");
-        AxisOptions vAxis = AxisOptions.create();
-        vAxis.set("format", "# Mb");
-        loadOpts.setVAxisOptions(vAxis);
 
-        loadOpts.setLegend(LegendPosition.RIGHT);
-        loadOpts.setColors("#fcaf3e", "#35a849", "#3a668d");
-        loadTable.setColumnLabel(1, "Used");
-        loadTable.setColumnLabel(2, "Free");
-        loadTable.setColumnLabel(3, "Total");
+        setYAxesTicksSuffix(" Mb");
+
+        setColors("#fcaf3e", "#35a849", "#3a668d");
+        //        loadOpts.setLegend(LegendPosition.RIGHT);
```

Comment:
you will remove this commented code in next PR as well. Right?

Reply:
Removed!